adoc/known-issues.adoc

```diff
@@ -47,6 +47,6 @@ You can safely ignore this warning.
 [#issue-update-apparmor]
 === AppArmor error messages in log after upgrade
 
-SLE Micro supports SELinux as the security framework, however, some AppArmor packages are still included because of package dependencies.
-Since they have been reduced since SLE Micro 5.1, it may happen that there are error messages showing in the system journal after upgrade.
+Leap Micro supports SELinux as the security framework, however, some AppArmor packages are still included because of package dependencies.
+Since they have been reduced since Leap Micro 5.1, it may happen that there are error messages showing in the system journal after upgrade.
```

Comment:
```suggestion
Since they have been reduced since {producta} 5.1, it may happen that there are error messages showing in the system journal after upgrade.
```